.github/ISSUE_TEMPLATE/feature_request.yml

```
@@ -0,0 +1,44 @@
+name: Feature Request
+description: Suggest an idea to enhance the current functionality of PyCytominer.
+title: "FeatRequest: "
+labels: [enhancement]
+
+body:
+  - type: checkboxes
+    id: checks
+    attributes:
+      label: Feature type
+      description: Please check the feature type you would like to propose.
+      options:
+        - label: >
+            Add new functionality
+        - label: >
+            Change existing functionality
+  - type: textarea
+    id: description
+    attributes:
+      label: General description of the proposed functionality
+      description: >
+        Please *concisely* describe what the proposed functionality is, if it is due to an existing issue, and how it will improve PyCytominer.
+    validations:
+      required: true
+  - type: textarea
+    id: feature-example
+    attributes:
+      label: Feature example
+      description: >
+        Please provide an example of this functionality (e.g., psuedocode).
+    validations:
+      required: true
+  - type: textarea
+    id: alternative-ideas
+    attributes:
+      label: Alternative Solutions
+      description: >
+        If applicable, please provide any alternative solutions/ideas for this functionality.
```

```suggestion
        If applicable, please suggest and/or provide any alternative solutions/ideas for the proposed functionality.
```
In this part, are we expecting a full detailed implementation of an alternative solution or are expecting a general idea?

The reason I am asking this is because of the words `provide` and `idea`

Provide -> full implementation of solving the problem (step by step code) 
suggest -> general `idea` that solving problem ( potentially no is code required) 

We can also use both words, making it welcoming for both parties (people with solutions and ideas). We can use all the help :D !

---

I was not thinking of full implementation but just an indication that someone is thinking about multiple ways to solve their issue, including their main solution.

I will change it to `suggest` to make it sound more optional, other than this section is optional and doesn't need to be filled out to submit the issue.

Thank you for bringing this up! I agree the language should be more clear in this case 😄